app/src/test/kotlin/fr/free/nrw/commons/contributions/ContributionsPresenterTest.kt

```diff
@@ -60,17 +60,6 @@ class ContributionsPresenterTest {
         liveData=MutableLiveData()
     }
 
-    /**
-     * Test fetch contributions
-     */
-    @Test
-    fun testFetchContributions(){
```

Comment:
This functionality hasn't been deleted just moved, right? It would be a shame to lose the coverage

Reply:
The functionality has changed but I have added a new test case for this method. I will add more test cases once the functionality gets tested.